internal/build/docker.go

```diff
@@ -176,10 +172,46 @@ func (l *localDockerBuilder) PushDocker(ctx context.Context, ref reference.Named
 	return reference.WithDigest(tag, pushedDigest)
 }
 
+// TODO(nick): Unify this with buildFromDfWithFiles
+func (l *localDockerBuilder) buildWithDfOnly(ctx context.Context, df Dockerfile) (digest.Digest, error) {
+	span, ctx := opentracing.StartSpanFromContext(ctx, "daemon-buildWithDfOnly")
+	defer span.Finish()
+
+	archive, err := TarWithDfOnly(ctx, df)
+	if err != nil {
+		return "", err
+	}
+
+	imageBuildResponse, err := l.dcli.ImageBuild(
```

Comment:
An important, currently-unplanned task that I think is necessary for Replicated is streaming the output of the build steps.

Does this change preclude that? Does this API allow streaming the output of the running commands, as they run?

Reply:
yep, that shouldn't change